amf-client/shared/src/test/resources/maker/declared-types.json

```diff
@@ -3,9 +3,6 @@
   "info":{
     "title": "test types",
     "description": "empty api only for test types"
-  },
-  "paths": {
```

Comment:
Why is the required paths facet removed here? Changes the api to an invalid definition.

Reply:
added `withEndpoints(empty)` in test